### ISSUE TITLE
Copy to clipboard not working for interactive examples

### ISSUE THREAD

**Issue by tannerdolby** [mdn/yari#10909]
### Summary

The copy-to-clipboard logic is failing in the interactive example `<iframe>` due to lack of clipboard-write permissions for the Clipboard API.

> To use the API in iframes, you need to enable it with [Permissions Policy](https://developer.chrome.com/docs/privacy-sandbox/permissions-policy), which defines a mechanism that allows for selectively enabling and disabling various browser features and APIs. Concretely, you need to pass either or both of clipboard-read or clipboard-write, depending on the needs of your app.

- https://web.dev/articles/async-clipboard#permissions_policy_integration

### URL

- https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path
- any page with an interactive example

### Reproduction steps

1. Go to https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path
2. Click into one of the clip-path demo values, then click the copy-to-clipboard icon
3. See error in console regarding lack of permissions
<img width="1277" alt="Screen Shot 2024-04-13 at 1 50 05 AM" src="https://github.com/mdn/yari/assets/48612525/f3f85b70-6d84-4046-89d6-c1b1ec8be648">

### Expected behavior

Expected content to be copied to clipboard within interactive examples

### Actual behavior

Fails to copy content to clipboard and logs error to the browser console

### Device

Desktop

### Browser

Chrome

### Browser version

Stable

### Operating system

Mac OS

### Screenshot

https://github.com/mdn/yari/assets/48612525/a50bec8a-1e85-434f-8332-28ac567ec24f

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).

**Comment by skyclouds2001** [comment]
maybe we could make the {{EmbedInteractiveExample}} macros able to passing a parameter to specified `allow` attribute on the `<iframe>` element, just like the {{EmbedLiveSample}} does

**Comment by tannerdolby** [comment]
@skyclouds2001 ya I think that's a good idea, since the EmbedLiveSample macro supports a parameter for allowed features ($6) it makes sense that the EmbedInteractiveExample macro would do the same. I added `allow="clipboard-write"` directly to the `<iframe>` in EmbedInteractiveExample.ejs for my initial PR (#10910) to quickly validate that the copy-to-clipboard worked again in interactive examples without having to update the actual EmbedInteractiveExample() macro calls throughout the various pages.

I searched mdn/content and there's 1256 results in 1253 files of `EmbedInteractiveExample(` which would mean doing a mass search-and-replace to add a third argument with the "clipboard-write" permission, that is assuming all the interactive examples have code snippets which can be copied to clipboard. If the mdn folks are cool with this work adding an extra parameter to the EmbedInteractiveExample macro to account for allowed features and updating those 1253 files in mdn/content then I will totally update my PR here to reflect that and then submit one in mdn/content.
